db/migrate/20170905111153_create_user_agents.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUserAgents < ActiveRecord::Migration
+  def change
+    create_table :user_agents do |t|
+      t.string :user_agent,           default: "", :limit => 1000
+      t.timestamps null: false
```

Comment:
I don't think this table needs timestamps. All we're doing is recording the time the user agent was first seen, which we can find from the `EventLog`.